### ISSUE TITLE
Spacing before stage directions

### ISSUE THREAD

**Issue by martindholmes** [projectLEMDO/lemdoIssues#42]
Peter C points out that in this speech:

emdFV_M.html#emdFV_M_s1_sp19

there is too much space between the end of the speech and the stage directions. Can't easily figure out why yet.

**Comment by martindholmes** [comment]
Assigning back to myself to check this before acting on it, because it looks like it may have been caused by lots of old tln lb elements that were causing FV_M to render badly.

**Comment by martindholmes** [comment]
This is not fixed by removing the extraneous lb elements, so more work to do here.

**Comment by martindholmes** [comment]
@JanelleJenstad I believe this layout issue -- the difference between the inter-speech spacing in Sel_M and FV_M -- is caused by the fact that FV speeches are all in prose and therefore all contain p elements. I've committed a fix in rev 9992 which solves this. We may want to consider adding more spacing between speeches, though, whether they're verse or prose.

Once we've confirmed that Sel_M and FV_M are both displaying with identical spacing, this one can be closed.
